packages/client/hmi-client/src/services/goLLM.ts

```diff
@@ -0,0 +1,53 @@
+import API from '@/api/api';
+import useAuthStore from '@/stores/auth';
+import { TaskStatus } from '@/types/Types';
+import { logger } from '@/utils/logger';
+import { EventSourceMessage, fetchEventSource } from '@microsoft/fetch-event-source';
+
+export async function modelCard(documentId: string, modelId: string): Promise<any> {
+	const response = await API.post('/gollm/model_card', null, {
+		params: {
+			'document-id': documentId,
+			'model-id': modelId
+		}
+	});
+
+	return response.data ?? null;
+}
+
+export async function handleTaskById(id: string, successHandler: (message: any) => void) {
```

Comment:
Could you refactor most of this function in the the API.ts utils `/src/api/api.ts` to be reuse easily by other SSE listener.